config/config.go

```diff
@@ -57,6 +57,7 @@ var (
 		LocalAstro:           newCfg("local.astrohub", "http://localhost:8871/v1"),
 		LocalCore:            newCfg("local.core", "http://localhost:8888/v1alpha1"),
 		LocalPublicAstro:     newCfg("local.public_astrohub", "http://localhost:8871/graphql"),
+		LocalPublicRESTAstro: newCfg("local.public_rest_astro", "http://localhost:8888/v1alpha1"),
```

Comment:
This seems same as LocalCore on line 58. Does core-api have multiple endpoints?

Reply:
Not today. I missed LocalCore. Will update the PR to use that instead of adding a new entry.